test/test_helper.bash

```diff
@@ -5,3 +5,20 @@ suts=$BATS_TEST_DIRNAME/suts
 function p {
     bash "$(dirname $BATS_TEST_DIRNAME)"/libexec/p "$@"
 }
+
+function init {
+    touch $suts/temp
+    echo "use_frameworks!
+platform :ios, '10.0'
+inhibit_all_warnings!
+
+target 'Altel' do
+  pod 'R.swift'
+  pod 'SwiftFormat/CLI'
```

Comment:
I think we can remove this pod right here, it doesn't help us in any way

Reply:
I left this pod to make sure that slash symbol doesn’t interfere with the function. I'll remove it

Reply:
@nurbekismagulov btw, very nice point! I didn't even think about it. Keep it then, if you feel like so